raco/fakedb.py

```diff
@@ -269,6 +269,15 @@ def limit(self, op):
         it = self.evaluate(op.input)
         return itertools.islice(it, op.count)
 
+    def orderby(self, op):
+        it = self.evaluate(op.input)
+        oList = zip(op.sort_columns, op.ascending)
+        sortedList = list(it)
+        for o in oList:
```

Comment:
This is the right approach for multikey sorting given a stable sort, but you need to sort in the opposite order of the given key: from least significant to most significant attribute. Therefore you just need to say
```
oList = reversed(zip(op.sort_columns, op.ascending))
```
There clearly needs to be a multikey ORDER BY test that would have caught this bug.